projects/experimentdao/index.js

```diff
@@ -0,0 +1,19 @@
+const retry = require('async-retry')
+const utils = require('../helper/utils');
+
+async function getData() {
+  const res = await retry(async bail => await utils.fetchURL('https://vite-api.thomiz.dev/tvl/experimentdao'))
```

Comment:
Can this be calculated using on-chain calls instead? We've got a guide on how to use our SDK here: https://docs.llama.fi/list-your-project/submit-a-project
 
 Please let me know if you need some help

Reply:
I don't really know, the assets are held in separate chains, would be a lot to implement in the adapter. I don't know if the sdk allows non-evm chains like Wax and XLM to be queried. 

So far, we're using:
 - Terra
 - BSC
 - Wax
 - Avalanche
 - Fantom
 - Harmony
 - XLM
 
And also, LP tokens from BSC/Harmony bridged to Vite don't really have a price, but the api takes the value of 1 lp token (in the redeemable form) as a price. I don't really know if the sdk can take in account this, AFAIK it uses coingecko ? 
 
I just figured out an api would be faster and easier to set up, as well as be easier for me to add new chains as soon as we integrate them. But if you think it's possible to make on-chains call directly to Wax/XLM/Terra, I might try. I never tried the SDK so I don't really know